evm-ds/build.rs

```diff
@@ -3,13 +3,28 @@ extern crate protoc_rust;
 fn main() {
     protoc_rust::Codegen::new()
         .out_dir("src/protos")
-        .inputs(&["protos/ScillaMessage.proto"])
-        .include("protos")
+        .inputs(&["../src/libPersistence/ScillaMessage.proto"])
+        .include("../src/libPersistence")
         .customize(protoc_rust::Customize {
             carllerche_bytes_for_bytes: Some(true),
             carllerche_bytes_for_string: Some(true),
             ..Default::default()
         })
         .run()
-        .expect("Running protoc failed.");
+        .expect("Running protoc failed for ScillaMessage.proto");
+
+    protoc_rust::Codegen::new()
+        .out_dir("src/protos")
```

Comment:
Generally I've seen the convention to generate code into `$OUT_DIR/protos.rs` and the `include!(concat!(env!("OUT_DIR), "/protos.rs"));` in your source (e.g. https://doc.rust-lang.org/cargo/reference/build-script-examples.html#code-generation)